### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by aurbiztondo-splunk**
Migrated TS docs, as per https://splunk.atlassian.net/browse/O11YDOCS-6111.

@jvoravong, can you approve?

### CODE REVIEW

**Comment by aurbiztondo-splunk** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -1,395 +1,19 @@
...
-    - Upgrading Kubernetes to at least 1.22.9 fixed the missing container
-      memory and pod network metrics.
-    - Upgrading containerd to at least 1.4.13 or 1.5.0 fixed the missing pod
-      memory metrics.
-  </details>
-- Kubernetes 1.23.0-1.23.6 using containerd - Memory stats/metrics can be
-  missing
-  <details>
-  <summary>Expand for more details</summary>
+For instructions on how to contribute to the docs, see [Contribute to Splunk Observability Cloud documentation]([https://docs.splunk.com/observability/en/get-started/contribute.html).
```
Comment (jvoravong) [inline docs/troubleshooting.md:19]: The precommit test should pass after this change.
```suggestion
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -1,395 +1,19 @@
...
+* [Troubleshoot the Collector for Kubernetes](https://docs.splunk.com/observability/en/gdi/opentelemetry/collector-kubernetes/troubleshoot-k8s.html)
+* [Troubleshoot the Collector for Kubernetes containers](https://docs.splunk.com/observability/en/gdi/opentelemetry/collector-kubernetes/troubleshoot-k8s-container.html)
 
-Even if you didn't provide enough resources the OTel Collector containers, under
-normal circumstances collector should not run out of memory. OOM can happen only
-if the collector is heavily throttled by the backend and exporter sending queue
-growing faster than collector can control memory utilization. In that case you
-should see 429 errors for metrics and traces or 503 errors for logs. Similar to
-the following:
+For general troubleshooting of the Splunk Distribution of the OpenTelemetry Collector see [Splunk OpenTelemetry Collector troubleshooting documentation]([https://github.com/signalfx/splunk-otel-collector/blob/main/docs/troubleshooting.md](https://docs.splunk.com/observability/en/gdi/opentelemetry/troubleshooting.html).
```
Comment (jvoravong) [inline docs/troubleshooting.md:8]:
```suggestion
For general troubleshooting of the Splunk Distribution of the OpenTelemetry Collector see [Splunk OpenTelemetry Collector troubleshooting documentation](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/troubleshooting.md](https://docs.splunk.com/observability/en/gdi/opentelemetry/troubleshooting.html).
```

**Comment by aurbiztondo-splunk** [comment]
Thanks for catching those, @jvoravong!

**Comment by aurbiztondo-splunk** [comment]
@jvoravong still having issues...

**Comment by jvoravong** [comment]
Flaky test, rerun fixed it.

**Pull request merged**